src/Stats.php

```diff
@@ -0,0 +1,227 @@
+<?php
+
+namespace ChrisJP\TTS;
+
+/**
+ * Methods to calculate various stats
+ * 
+ */
+class Stats extends TTS
+{
+
+    /**
+     * An array of every file in AUDIO_DIR with the .mp3 extension
+     *
+     * @var array
+     */
+    private $audioFileArray = [];
```

Comment:
It may be good to add type hints to these: `private array $audioFileArray = [];`